pegasus/src/database.rb

```diff
@@ -31,8 +31,8 @@ def initialize(table, no_cache = false)
       DB[@table].select(*@column_aliases).all.to_json
     end.deep_dup
 
-    @contents = JSON.parse(json_contents, object_class: DB[@table]).each do |tutorial|
-      tutorial.values.symbolize_keys!
+    @contents = JSON.parse(json_contents, object_class: DB[@table]).map do |tutorial|
+      tutorial.instance_values['values'].symbolize_keys!
```

Comment:
Can you explain a little bit more how this change is changing the behavior for the CdoTutorials class? I don't think I understand what this edit would do.

Reply:
This means that we're not storing the class object itself in the cache, instead storing a hash of properties. This is based on my theory -- which could be incorrect -- that storing the dynamic class in the cache was leading to these problems.